app/routes/models/enrich-request.js

```diff
@@ -0,0 +1,85 @@
+function ViewModel (error) {
+  const errorMessage = { }
+  if (error) {
+    for (const detail of error.details) {
+      detail.text = detail.message
+      errorMessage[detail.context.label] = 'govuk-input--error'
+      if (detail.context.label === 'day' ||
+        detail.context.label === 'month' ||
+        detail.context.label === 'year') {
+        detail.href = `#debt-discovered-date-${detail.context.label}`
+        errorMessage['date-error'] = {
+          text: 'The date submitted is not valid'
+        }
+      } else {
+        detail.href = `#${detail.context.label}`
+        errorMessage['radio-error'] = {
+          text: 'Select the type of debt'
+        }
+      }
+    }
+
+    errorMessage.summary = {
+      titleText: 'There is a problem',
+      errorList: error.details
+    }
+  }
+
+  this.model = {
+    errorMessage: errorMessage.summary,
+    radio: {
+      id: 'debt-type',
+      name: 'debt-type',
+      classes: 'govuk-radios--small',
+      fieldset: {
+        legend: {
+          text: 'Type of debt',
+          classes: 'govuk-label govuk-label--s'
+        }
+      },
+      items: [
+        {
+          value: 'irregular',
+          text: 'Irregular'
+        },
+        {
+          value: 'administrative',
+          text: 'Administrative'
+        }
+      ],
+      errorMessage: errorMessage['radio-error']
+    },
+    date: {
+      id: 'debt-discovered-date',
+      fieldset: {
+        legend: {
+          text: 'Debt date discovered',
+          classes: 'govuk-fieldset__legend govuk-fieldset__legend--s'
+        }
+      },
+      hint: {
+        text: 'For example, 27 3 2007'
+      },
+      errorMessage: errorMessage['date-error'],
+      items: [
```

Comment:
Is there a reason for the default data been set. Currently it is set to 06/03/2076

Also does there need to be a minimum year the user can set?

Reply:
Completely missed that, changed.
As for the year, I thought it would be good to have 4 digits all the time but can remove the minimum year validation. Will leave the maximum one cause the datatype for the column is VARCHAR(10) so it would go over the limit if it's more than 9999